src/Layers/xrRenderPC_GL/rgl.cpp

```diff
@@ -449,7 +449,7 @@ void CRender::OnFrame()
 void CRender::OnFrame()
 {
     Models->DeleteQueue();
-    if (ps_r2_ls_flags.test(R2FLAG_EXP_MT_CALC))
+    if (ps_r2_ls_flags.test(R2FLAG_EXP_MT_CALC) && Details)
```

Comment:
You just disabled HOM. =(

Please, just remove this, this should be fixed in other way.
`Details` can't be nullptr here. If it is, then error is in another place.